spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -227,6 +232,73 @@ module Oauth
       expect(response).to have_http_status :forbidden
     end
 
+    it "should let an admin create an application with Oauth Admins" do
+      controller.sign_in! admin
+      post(:create,
+        params: {
+            doorkeeper_application: {
+              name: 'Some app',
+              redirect_uri: 'https://www.example.com',
+              can_message_users: true,
+            },
+            member_ids: user2.id
```

Comment:
Is there a spec somewhere that exercises the "number space number space" member IDs input?  If not, we need one.

Reply:
Added test